src/Lib/Core/Jwt.hs

```diff
@@ -37,11 +38,11 @@ jwtPayloadFromMap claimsMap = do
   jwtUserId <- UUID.fromText jwtId
   return JWTPayload{..}
 
-mkJWTToken :: (MonadIO m, MonadReader AppEnv m) => Int -> JWTPayload -> m Text
-mkJWTToken expiryInSeconds payload = do
+mkJWTToken :: (MonadIO m, MonadReader AppEnv m) => Seconds -> JWTPayload -> m Text
+mkJWTToken (Seconds expiry) payload = do
```

Comment:
I just noticed one more thing. This `JWT` part is not finished.

Let's perform more renaming here as well:

1. `mkJWTToken` -> `mkJwtToken`
2. `JWTPayload` -> `JwtPayload`
3. `decodeAndVerifyJWTToken` -> `decodeAndVerifyJwtToken`

Also, we need to introduce `JwtSecret` and `JwtToken` data types:

https://github.com/Holmusk/CDMP/blob/cd24a6fb3f3d187f68bdd0d8ff7056ba456f5f58/backend/src/CDMP/Core/Jwt.hs#L33-L38

`JwtSecret` will be stored in `AppEnv`.
`mkJwtToken` function will return `JwtToken` instead of `Text`
`decodeAndVerifyJwtToken` will take `JwtToken` instead of `Text`.